api/src/main/resources/marquez/db/migration/V54__lineage_events_created_at_indexed.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE lineage_events ADD created_at TIMESTAMP;
```

Comment:
`TIMESTAMP WITH TIME ZONE`. I think it's good practice to just always include the time zone in the timestamp.

Reply:
Thanks, i kept it as TIMESTAMP following other created_at columns in the schema.  But i will update this one to TIMESTAMP WITH TIME ZONE. 

Additionally , i was before backfilling old entries with event_time. I have removed that step now because it might be a costly operation during migration.

Reply:
updated the type to TIMESTAMP WITH TIME ZONE